content/docs/operating/configuration.md

```diff
@@ -309,18 +309,21 @@ feature to replace the special `__address__` label.
 
 ### `<dns_sd_config>`
 
-A DNS-SD configuration allows specifying a set of DNS record names which
-are periodically queried to discover a list of targets (host-port pairs). The
-DNS servers to be contacted are read from `/etc/resolv.conf`.
+A DNS configuration allows specifying a set of DNS domain names which are
+periodically queried to discover a list of targets. The DNS servers to be
+contacted are read from `/etc/resolv.conf`.
 
-During the [relabeling phase](#relabel_config), the meta
-label `__meta_dns_name` is available on each target and is set to the SRV
+This service discovery method only supports basic DNS queries, but not the
```

Comment:
Maybe "only supports basic DNS A-record and SRV-record queries" so that people don't incorrectly assume from this paragraph that SRV lookups aren't possible at all.